core/tauri-runtime-wry/src/lib.rs

```diff
@@ -117,6 +118,8 @@ type IpcHandler = dyn Fn(String) + 'static;
 mod webview;
 pub use webview::Webview;
 
+const RESIZE_INTERVAL: Duration = Duration::from_millis(1000 / 120);
```

Comment:
is there a reason for this specific interval? or should we get it from monitor refresh rate?

Reply:
Anything faster was getting quite laggy in my testing especially times 4 for the multiwebview example.

But making it variable is still useful, at least to get it lower than 120 for slower displays.